plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -12,6 +12,7 @@
 
 use Automattic\Jetpack\Constants;
 use Automattic\WooCommerce\Enums\PaymentGatewayFeature;
+use Automattic\WooCommerce\Gateways\PayPal\Buttons;
```

Comment:
Suggestion (here and in the other files where this is included) for increased code readability:

```php
use Automattic\WooCommerce\Gateways\PayPal\Buttons as PayPalButtons;
```

This way anyone going throught the code won't be wondering "what kind of buttons are those?" before going to the top  of the file.

Reply:
Good idea. Applied it in https://github.com/woocommerce/woocommerce/pull/61204/commits/923b5bb215cd5509bf32340f7b64cb7629f5e216